### PR TITLE
scoreboard tests

### PR DESCRIPTION
**Pull request by kashyapgajera**
IO tests

### CODE REVIEW

File: tests/functional_tests/helper_scripts/scoreboard/worse_score1.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
```
Comment (prasadtalasila) [inline tests/functional_tests/helper_scripts/scoreboard/worse_score1.sh]: file name is not intuitive. May be worse_score_setup.sh

File: tests/functional_tests/helper_scripts/scoreboard/worse_score2.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```
Comment (prasadtalasila) [inline tests/functional_tests/helper_scripts/scoreboard/worse_score2.sh]: file name not intuitive. May be worse_score_restore.sh

File: tests/functional_tests/helper_scripts/scoreboard/worse_score1.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+#copies incorrect code file Buyer_mistake from data/scoreboard and places it in student solution
```
Comment (prasadtalasila) [inline tests/functional_tests/helper_scripts/scoreboard/worse_score1.sh]: #!/bin/bash

File: tests/functional_tests/scoreboard.bats [hunk 4]
```diff
@@ -13,41 +15,93 @@ teardown() {
...
+	result=$?
+	rm -rf $BATS_TMPDIR/scoreboard
+	[ "$result" -eq 0 ]
 }
 
 @test "no change on scoreboard for worse score" {
-		# check for the correctly updated scoreboard
-		skip "TODO"
+	node submit.js -i 2015A7PS006G -l lab1 --lang=java --host='localhost:9000' >/dev/null
+	bash ./helper_scripts/scoreboard/worse_score1.sh
```
Comment (prasadtalasila) [inline tests/functional_tests/scoreboard.bats]: appropriate file name changes for worse_score1.sh and worse_score2.sh as suggested before.

**Comment by kashyapgajera** [comment]
i made the necessary changes

**Pull request merged**